engine/routines.go

```diff
@@ -287,9 +287,15 @@ func WebsocketDataHandler(ws *wshandler.Websocket) {
 				log.Errorf(log.WebsocketMgr, "routines.go exchange %s websocket error - %s", ws.GetName(), data)
 			case wshandler.TradeData:
 				// Trade Data
-				// if Bot.Settings.Verbose {
-				//	log.Println("Websocket trades Updated:   ", data.(exchange.TradeData))
-				// }
+				if Bot.Settings.Verbose {
+					log.Infoln(log.Global, "Websocket trades Updated: ", d)
```

Comment:
Please copy this format here: https://github.com/thrasher-corp/gocryptotrader/blob/master/engine/routines.go#L339 and use %+v to display the data for both trade/funding updates up until this gets fleshed out